ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v1/container/clientlibs/editor/js/editDialog.js

```diff
@@ -42,6 +42,259 @@
 
         Utils = window.CQ.FormsCoreComponents.Utils.v1;
 
```

Comment:
please make sure that things work from form editor side as well. Since this change will be reflected in form container, authors will be able to use it via form editor as well.

Reply:
veified